.github/workflows/test.yml

```diff
@@ -0,0 +1,43 @@
+name: "Run Test and Package Latest JS"
+
+on:
+  push:
+    branches:
+
+jobs:
+  unit-tests:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - run: npm ci
+    - run: npm test
+
+  check-latest-dist:
+    runs-on: ubuntu-latest
+    needs: [unit-tests]
+    steps:
+    - uses: actions/checkout@v2
+    - run: npm install
+    - run: yarn prepare
```

Comment:
I think a `name` here would be really useful to call this out as the step that builds the compiled action

Reply:
Sure thing 👍